client/my-sites/customer-home/cards/education/blogging-quick-start/index.jsx

```diff
@@ -19,13 +23,13 @@ const BloggingQuickStart = () => {
 				{
 					ModalComponent: BloggingQuickStartModal,
 					modalComponentProps: {
-						isVisible: isModalVisible,
+						isVisible: isModalOpen,
 						onClose: ( event ) => {
 							event.preventDefault();
-							setIsModalVisible( false );
+							closeModal();
 						},
 					},
-					onClick: () => setIsModalVisible( true ),
+					onClick: () => openModal(),
```

Comment:
Super duper nit:

```suggestion
					onClick: openModal,
```

To avoid passing functions as props, and to make it clearer 😄

Reply:
If other places pass this function to the button directly, it will be called with event argument such as `openModal( event )`. So I prefer keeping this way.

Reply:
Ah right right sorry, forgot about this `event` arg 🤦‍♂️